packages/venia-ui/lib/tokens.css

```diff
@@ -0,0 +1,133 @@
+/* global tokens */
+:global(:root) {
+    /* animation */
+    --venia-global-anim-bounce: cubic-bezier(0.5, 1.8, 0.9, 0.8);
+    --venia-global-anim-in: cubic-bezier(0, 0, 0.2, 1);
+    --venia-global-anim-out: cubic-bezier(0.4, 0, 1, 1);
+    --venia-global-anim-standard: cubic-bezier(0.4, 0, 0.2, 1);
+
+    /* color */
+    --venia-global-color-blue-400: 61, 132, 255;
+    --venia-global-color-blue-500: 51, 109, 255;
+    --venia-global-color-blue-600: 41, 84, 255;
+    --venia-global-color-blue-700: 31, 57, 255;
+    --venia-global-color-gray-50: 255, 255, 255;
+    --venia-global-color-gray-75: 250, 250, 250;
+    --venia-global-color-gray-100: 244, 245, 245;
+    --venia-global-color-gray-200: 234, 235, 235;
+    --venia-global-color-gray-300: 223, 225, 226;
+    --venia-global-color-gray-400: 181, 184, 186;
+    --venia-global-color-gray-500: 149, 154, 157;
+    --venia-global-color-gray-600: 118, 123, 127;
+    --venia-global-color-gray-700: 84, 93, 99;
+    --venia-global-color-gray-800: 51, 63, 71;
+    --venia-global-color-gray-900: 23, 32, 38;
+    --venia-global-color-gray: var(--venia-global-color-gray-100);
+    --venia-global-color-gray-dark: var(--venia-global-color-gray-300);
+    --venia-global-color-gray-darker: var(--venia-global-color-gray-600);
+    --venia-global-color-green-400: 51, 171, 132;
+    --venia-global-color-green-500: 45, 157, 120;
+    --venia-global-color-green-600: 38, 142, 108;
+    --venia-global-color-green-700: 18, 128, 92;
+    --venia-global-color-orange: 241, 99, 33;
```

Comment:
The orange property is used only for the toast area, and have left this as is, since there's no option for that as of yet in the new style